docs/04-operation-guides/troubleshooting/01-deplyoment-fails-pending-pods.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Local Kyma deployment fails with pending Pods
+---
+
+## Symptom
+
+Local Kyma deployment with k3d fails with one or more Pods in pending state.
+Describing the pod reveals an error message like:
+
+`0/2 nodes are available: 2 node(s) had taint {node.kubernetes.io/disk-pressure: }, that the pod didn't tolerate.`
+
+## Cause
+
+The underlying Docker environment ran out of resources (memory/cpu/disk). 
+Thus, k3d marked all Kubernetes nodes with a taint "disk-pressure".
+
+## Remedy
+
+Verify the cause:
+
+1. Find out which Pods are pending.
+2. For the pending Pods, verify which error message you get by _describing_ the related deployment:
+   For example, if the Istiod Pod is pending, run `kubectl -n istio-system describe deployment istiod` and `kubectl -n istio-system describe pod istiod-{POD_NAME}`.
+
+Fix the issue:
+
+1. In the preference settings of your Docker UI, adjust the Docker resource assignment.
```

Comment:
I'd rather avoid click-level instructions if possible.

Reply:
Sure, it's just the `preference settings` that stroke me off as redundant/duplication. I'd use either `preferences` or `settings`. 
Also, technically, everything that the user can interact with is a UI, so Docker UI does not necessarily mean Docker Desktop, it could also be via Terminal, that's why I used the precise name.

Reply:
Now I see. Thanks for following up!

How about:

```suggestion
1. In the preferences of your Docker Desktop, adjust the Docker resource assignment.
```